docs/modules/deployment/pages/restore/centos-rhel8/restore-centos-rhel8.adoc

```diff
@@ -0,0 +1,105 @@
+[[restore-centos-rhel8]]
+.Restore User/Group definition
+
+[source, console]
+----
+sudo cat /tmp/opennms-passwd.txt >> /etc/passwd
+sudo cat /tmp/opennms-group.txt >> /etc/group
+----
+
+.Create log folder
+----
+sudo mkdir /var/log/opennms
+chown opennms. /var/log/opennms
+----
+
+.Restore OpenNMS {page-component-title} binaries, RRD archives, configurations
+
+[source, console]
+----
+sudo tar -xzf /tmp/opennms-opt.tar.gz -C / && \
+sudo tar -xzf /tmp/opennms-var.tar.gz -C /
+----
+
+.Create OpenNMS systemd unit
+[source, console]
+----
+sudo cp /opt/opennms/etc/opennms.service /lib/systemd/system/
+----
+
+._Assign CAP_NET_RAW capabilities_
```

Comment:
@mmahacek Can we include this part from the core installation somehow? I mean in an embedded way.

Reply:
Referencing now to the install guide, to avoid duplication.